crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -174,12 +172,12 @@ class C:
 reveal_type(C.pure_class_variable1)  # revealed: str
 
 # TODO: this should be `Literal[1]`, or `Unknown | Literal[1]`.
-reveal_type(C.pure_class_variable2)  # revealed: @Todo(Unsupported or invalid type in a type expression)
+reveal_type(C.pure_class_variable2)  # revealed: Unknown
```

Comment:
This is another case that was arguably better before with the `@Todo`.

Reply:
This is just because we don't yet support inferring the type when we encounter a bare-ClassVar annotation? That seems pretty easy to fix, so I'm not worried about this.